--- lxc-lxc-1.0.0.alpha3/configure.ac~	2013-11-15 23:08:17.000000000 +0200
+++ lxc-lxc-1.0.0.alpha3/configure.ac	2013-11-20 01:01:35.253197161 +0200
@@ -81,7 +81,7 @@
 
 if test "x$enable_doc" = "xyes" -o "x$enable_doc" = "xauto"; then
 	db2xman=""
-	dbparsers="docbook2x-man db2x_docbook2man docbook2man"
+	dbparsers="docbook2X2man docbook2x-man db2x_docbook2man docbook2man"
 
 	AC_MSG_CHECKING(for docbook2x-man)
 	for name in ${dbparsers}; do
@@ -525,6 +525,7 @@
 	templates/lxc-opensuse
 	templates/lxc-busybox
 	templates/lxc-fedora
+	templates/lxc-pld
 	templates/lxc-openmandriva
 	templates/lxc-oracle
 	templates/lxc-altlinux
--- lxc-lxc-1.0.0.alpha3/templates/Makefile.am~	2013-11-15 23:08:17.000000000 +0200
+++ lxc-lxc-1.0.0.alpha3/templates/Makefile.am	2013-11-20 01:04:32.994128379 +0200
@@ -6,6 +6,7 @@
 	lxc-ubuntu-cloud \
 	lxc-opensuse \
 	lxc-fedora \
+	lxc-pld \
 	lxc-openmandriva \
 	lxc-oracle \
 	lxc-altlinux \
