--- configure.ac.orig	2016-08-16 04:37:55.000000000 +0200
+++ configure.ac	2016-08-20 12:05:05.962758223 +0200
@@ -57,6 +57,7 @@
 	AC_CHECK_FILE(/etc/mandriva-release,with_distro="openmandriva")
 	AC_CHECK_FILE(/etc/pardus-release,with_distro="pardus")
 	AC_CHECK_FILE(/etc/altlinux-release,with_distro="altlinux")
+	AC_CHECK_FILE(/etc/pld-release,with_distro="pld")
 fi
 with_distro=`echo ${with_distro} | tr '[[:upper:]]' '[[:lower:]]'`
 
@@ -72,7 +73,7 @@
 		distroconf=default.conf.lxcbr
 		distrosysconf="$sysconfdir/default"
 		;;
-	redhat|centos|fedora|oracle|oracleserver|sparclinux|altlinux|suse|opensuse*|plamo)
+	redhat|centos|fedora|oracle|oracleserver|sparclinux|altlinux|suse|opensuse*|plamo|pld)
 		distroconf=default.conf.lxcbr
 		distrosysconf="$sysconfdir/sysconfig"
 		;;
@@ -108,6 +109,9 @@
 			ubuntu)
 				init_script=upstart,systemd
 				;;
+			pld)
+				init_script=sysvinit,upstart,systemd
+				;;
 			*)
 				echo -n "Linux distribution init system unknown."
 				init_script=
@@ -168,7 +172,7 @@
 
 if test "x$enable_doc" = "xyes" -o "x$enable_doc" = "xauto"; then
 	db2xman=""
-	dbparsers="docbook2x-man db2x_docbook2man docbook2man docbook-to-man"
+	dbparsers="docbook2X2man docbook2x-man db2x_docbook2man docbook2man docbook-to-man"
 
 	AC_MSG_CHECKING(for docbook2x-man)
 	for name in ${dbparsers}; do
@@ -834,6 +838,7 @@
 	templates/lxc-opensuse
 	templates/lxc-oracle
 	templates/lxc-plamo
+	templates/lxc-pld
 	templates/lxc-slackware
 	templates/lxc-sshd
 	templates/lxc-ubuntu
--- templates/Makefile.am.orig	2016-08-20 11:55:34.232752000 +0200
+++ templates/Makefile.am	2016-08-20 11:57:12.522753070 +0200
@@ -15,6 +15,7 @@
 	lxc-opensuse \
 	lxc-oracle \
 	lxc-plamo \
+	lxc-pld \
 	lxc-slackware \
 	lxc-sshd \
 	lxc-ubuntu \
