diff -durN -x '*~' -x '*.orig' lxc-1.1.1.orig/configure.ac lxc-1.1.1/configure.ac
--- lxc-1.1.1.orig/configure.ac	2015-03-16 20:57:47.000000000 +0100
+++ lxc-1.1.1/configure.ac	2015-04-02 14:40:25.000000000 +0200
@@ -63,7 +63,7 @@
 		distroconf=default.conf.lxcbr
 		distrosysconf="$sysconfdir/default"
 		;;
-	redhat|centos|fedora|oracle|oracleserver|suse|opensuse*)
+	redhat|centos|fedora|oracle|oracleserver|suse|opensuse*|pld)
 		distroconf=default.conf.lxcbr
 		distrosysconf="$sysconfdir/sysconfig"
 		;;
@@ -99,6 +99,9 @@
 			ubuntu)
 				init_script=upstart,systemd
 				;;
+			pld)
+				init_script=upstart,systemd
+				;;
 			*)
 				echo -n "Linux distribution init system unknown."
 				init_script=
@@ -152,7 +154,7 @@
 
 if test "x$enable_doc" = "xyes" -o "x$enable_doc" = "xauto"; then
 	db2xman=""
-	dbparsers="docbook2x-man db2x_docbook2man docbook2man docbook-to-man"
+	dbparsers="docbook2X2man docbook2x-man db2x_docbook2man docbook2man docbook-to-man"
 
 	AC_MSG_CHECKING(for docbook2x-man)
 	for name in ${dbparsers}; do
@@ -756,6 +758,7 @@
 	templates/lxc-opensuse
 	templates/lxc-oracle
 	templates/lxc-plamo
+	templates/lxc-pld
 	templates/lxc-sshd
 	templates/lxc-ubuntu
 	templates/lxc-ubuntu-cloud
diff -durN -x '*~' -x '*.orig' lxc-1.1.1.orig/templates/Makefile.am lxc-1.1.1/templates/Makefile.am
--- lxc-1.1.1.orig/templates/Makefile.am	2015-03-16 20:57:47.000000000 +0100
+++ lxc-1.1.1/templates/Makefile.am	2015-04-02 14:38:17.000000000 +0200
@@ -15,6 +15,7 @@
 	lxc-opensuse \
 	lxc-oracle \
 	lxc-plamo \
+	lxc-pld \
 	lxc-sshd \
 	lxc-ubuntu \
 	lxc-ubuntu-cloud
