--- lxc-3.2.1/src/lxc/cgroups/cgfsng.c.orig	2019-07-23 00:34:33.000000000 +0200
+++ lxc-3.2.1/src/lxc/cgroups/cgfsng.c	2020-03-24 15:17:05.536616935 +0100
@@ -496,12 +496,12 @@
 	}
 
 	if (!flipped_bit) {
-		DEBUG("No isolated or offline cpus present in cpuset");
-		return true;
+		cpulist = lxc_cpumask_to_cpulist(possmask, maxposs);
+		TRACE("No isolated or offline cpus present in cpuset");
+	} else {
+		cpulist = move_ptr(posscpus);
+		TRACE("Removed isolated or offline cpus from cpuset");
 	}
-	DEBUG("Removed isolated or offline cpus from cpuset");
-
-	cpulist = lxc_cpumask_to_cpulist(possmask, maxposs);
 	if (!cpulist) {
 		ERROR("Failed to create cpu list");
 		return false;
@@ -1260,7 +1260,7 @@
 
 	orig_len = strlen(dir);
 	do {
-		__do_free char *makeme;
+		__do_free char *makeme = NULL;
 		int ret;
 		size_t cur_len;
 
