diff -Nur lxc-0.6.2.orig/src/lxc/Makefile.am lxc-0.6.2.ldflags/src/lxc/Makefile.am
--- lxc-0.6.2.orig/src/lxc/Makefile.am	2009-04-24 03:43:33.000000000 -0600
+++ lxc-0.6.2.ldflags/src/lxc/Makefile.am	2009-05-03 14:50:02.000000000 -0600
@@ -1,7 +1,7 @@
 INCLUDES= -I$(top_srcdir)/src -DLXCPATH="\"@LXCPATH@\"" \
 	  -DLXCBINDIR="\"$(bindir)\"" \
 	  -DLXCLIBEXECDIR="\"$(libexecdir)\""
-AM_LDFLAGS= -lutil
+LDFLAGS= -lutil
 lib_LTLIBRARIES = liblxc.la
 pkginclude_HEADERS = \
 		error.h \
