--- lxc-0.9.0/configure.ac.orig	2013-04-05 11:47:10.000000000 +0200
+++ lxc-0.9.0/configure.ac	2013-04-17 20:04:27.905564169 +0200
@@ -67,7 +67,7 @@
 	db2xman=""
 
 	AC_MSG_CHECKING(for docbook2x-man)
-	for name in docbook2x-man db2x_docbook2man; do
+	for name in docbook2X2man docbook2x-man db2x_docbook2man; do
 		if "$name" --help >/dev/null 2>&1; then
 			db2xman="$name"
 			break;
@@ -140,6 +140,8 @@
 # Lua module and scripts
 if test x"$with_distro" = "xdebian" -o x"$with_distro" = "xubuntu" ; then
     LUAPKGCONFIG=lua5.1
+elif test x"$with_distro" = "xpld"; then
+    LUAPKGCONFIG=lua51
 else
     LUAPKGCONFIG=lua
 fi
--- lxc-0.9.0/templates/Makefile.am~	2013-03-19 00:17:01.000000000 +0200
+++ lxc-0.9.0/templates/Makefile.am	2013-11-17 03:44:48.326889371 +0200
@@ -6,6 +6,7 @@
 	lxc-ubuntu-cloud \
 	lxc-opensuse \
 	lxc-fedora \
+	lxc-pld \
 	lxc-oracle \
 	lxc-altlinux \
 	lxc-busybox \
