--- a/config/init/common/lxc-net.in	2022-11-21 23:35:46.490227758 +0100
+++ b/config/init/common/lxc-net.in	2022-11-21 23:36:44.836896223 +0100
@@ -27,6 +27,16 @@
 
 [ ! -f $distrosysconfdir/lxc ] || . $distrosysconfdir/lxc
 
+# Additional network based on macvlan
+# It can be overridden in @LXC_DISTRO_SYSCONF@/lxc_macvlan
+# by default is not used
+
+macvlan="@LIBEXECDIR@/lxc/lxc_macvlan"
+USE_LXC_MACVLAN="false"
+
+[ ! -f $distrosysconfdir/lxc_macvlan ] || . $distrosysconfdir/lxc_macvlan
+[ ! -f $macvlan ] || . $macvlan
+
 use_nft() {
     [ -n "$NFT" ] && nft list ruleset > /dev/null 2>&1 && [ "$LXC_USE_NFT" = "true" ]
 }
@@ -112,6 +122,7 @@
 }
 
 start() {
+    [ "x$USE_LXC_MACVLAN" = "xtrue" ] && { macvlan_start; exit $?; }
     [ "x$USE_LXC_BRIDGE" = "xtrue" ] || { exit 0; }
 
     [ ! -f "${varrun}/network_up" ] || { echo "lxc-net is already running"; exit 1; }
@@ -222,6 +233,7 @@
 }
 
 stop() {
+    [ "x$USE_LXC_MACVLAN" = "xtrue" ] && { macvlan_stop; exit $?; }
     [ "x$USE_LXC_BRIDGE" = "xtrue" ] || { exit 0; }
 
     [ -f "${varrun}/network_up" ] || [ "$1" = "force" ] || { echo "lxc-net isn't running"; exit 1; }
