diff -urN lxc-3.0.3.orig/src/lxc/log.c lxc-3.0.3/src/lxc/log.c
--- lxc-3.0.3.orig/src/lxc/log.c	2018-11-23 00:08:27.000000000 +0100
+++ lxc-3.0.3/src/lxc/log.c	2019-01-28 19:52:33.204408097 +0100
@@ -271,7 +271,11 @@
 	seconds = (time->tv_sec - d_in_s - h_in_s - (minutes * 60));
 
 	/* Make string from nanoseconds. */
+        #ifdef __ILP32__
+        ret = snprintf(nanosec, sizeof(nanosec), "%lld", (int64_t)time->tv_nsec);
+        #else
 	ret = snprintf(nanosec, sizeof(nanosec), "%"PRId64, (int64_t)time->tv_nsec);
+	#endif
 	if (ret < 0 || ret >= sizeof(nanosec))
 		return -1;
 
