--- lxc-1.0.7/src/lxc/log.c.orig	2014-12-05 21:50:20.000000000 +0100
+++ lxc-1.0.7/src/lxc/log.c	2015-03-13 21:30:29.354757660 +0100
@@ -91,7 +91,11 @@
 
 	ms = event->timestamp.tv_usec / 1000;
 	n = snprintf(buffer, sizeof(buffer),
+#ifdef __ILP32__
+		     "%15s %10lld.%03d %-8s %s - %s:%s:%d - ",
+#else
 		     "%15s %10ld.%03d %-8s %s - %s:%s:%d - ",
+#endif
 		     log_prefix,
 		     event->timestamp.tv_sec,
 		     ms,
