diff -ruNp lxc-4.0.7.orig/src/lxc/log.c lxc-4.0.7/src/lxc/log.c
--- lxc-4.0.7.orig/src/lxc/log.c	2021-04-29 22:57:15.000000000 +0200
+++ lxc-4.0.7/src/lxc/log.c	2021-04-30 17:29:22.505346892 +0200
@@ -280,7 +280,11 @@ static int lxc_unix_epoch_to_utc(char *b
 	seconds = (time->tv_sec - d_in_s - h_in_s - (minutes * 60));
 
 	/* Make string from nanoseconds. */
+        #ifdef __ILP32__
+        ret = snprintf(nanosec, sizeof(nanosec), "%lld", (int64_t)time->tv_nsec);
+        #else
 	ret = strnprintf(nanosec, sizeof(nanosec), "%"PRId64, (int64_t)time->tv_nsec);
+	#endif
 	if (ret < 0)
 		return ret_errno(EIO);
 
